root-packages/nexttrace-enhanced/build.sh

```diff
@@ -1,11 +1,11 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/OwO-Network/nexttrace-enhanced
+TERMUX_PKG_HOMEPAGE=https://github.com/nxtrace/Ntrace-V1
 TERMUX_PKG_DESCRIPTION="An open source visual routing tool that pursues light weight"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.3.2-beta.1"
+TERMUX_PKG_VERSION="1.2.0.1"
 TERMUX_PKG_REVISION=2
```

Comment:
Thanks for PR, please remove this line.

Reply:
I'm very sorry that I didn't notice this place before. I have modified it. Please review it.